src/main/java/redis/clients/jedis/util/Pool.java

```diff
@@ -33,6 +38,12 @@ public boolean isClosed() {
     return this.internalPool.isClosed();
   }
 
+  /**
+   * @param poolConfig
+   * @param factory
+   * @deprecated This could be private in future.
+   */
+  @Deprecated
   public void initPool(final GenericObjectPoolConfig poolConfig, PooledObjectFactory<T> factory) {
```

Comment:
Is it deprecated or just going to be private in the future?

Reply:
_just going to be private in the future_